docker-compose.yml

```diff
@@ -310,9 +310,9 @@ services:
     # Jaeger supports OTEL out of the box
     image: jaegertracing/opentelemetry-all-in-one:latest
     ports:
-      - 13133:13133  # health check port
-      - 16686:16686  # UI viewer
-      - 4317:55680   # Otel endpoint
+      - 13133:13133 # health check port
+      - 16686:16686 # UI viewer
+      - 4317:55680 # Otel endpoint
 
 networks:
-  databases:
+  databases: null
```

Comment:
I saw the same change on the engineer PR and should've asked before: what does this do?

Reply:
it's just the plugin autocompleting the correct key now. It was always implicitly null, just explicit now.